assets/TWO_VIEW_ESTIMATOR.md

```diff
@@ -8,4 +8,73 @@
 - **Two View Estimator**
 - [Multiview Optimizer](assets/MULTIVIEW_OPTIMIZER.md)
 
-## What is a Two-View Estimator?
\ No newline at end of file
+## What is a Two-View Estimator?
+
+
+Two-View Estimator (TVE) takes information about two images and tries to determine their relative pose (how one camera is positioned and oriented with respect to the other one). It also generates correspondences between keypoint in the images.
```

Comment:
remove: how one camera is positioned and oriented with respect to the other one

use 3D relative orientation and 3D unit translation (or translation direction) to avoid ambiguity.